react/features/app/actions.native.js

```diff
@@ -127,8 +129,11 @@ export function appNavigate(uri: ?string) {
         dispatch(setRoom(room));
 
         if (room) {
-            dispatch(createDesiredLocalTracks());
-            dispatch(connect());
+            navigateRoot(screen.preJoin);
```

Comment:
Check if the prejoin screen is enabled before navigating, if it's not, dispatch the actions inline here